test/system-test/test_auth.py

```diff
@@ -0,0 +1,47 @@
+from endpoints import auth
+from utils import apply_kms_constitution, trust_jwt_issuer
+
+
+def test_auth_member_cert(setup_kms):
+    status_code, auth_json = auth(auth="member_cert")
+    print(auth_json)
```

Comment:
I like this change to print the output of the endpoint call for tests it makes the output much clearer what happened, maybe we can put this print inside the call_endpoint wrapper so all tests benefit?

Reply:
This is already outputted in endpoints

Reply:
Ah awesome, so why are we printing it a second time just after it's been printed?